dist/api_data.py

```diff
@@ -1325,7 +1325,7 @@ def __ge__(self, other):
         behavior (for example, a leaked block)''',
         type='boolean')
 ]),
-
+'WT_SESSION.dump' : Method([]),
```

Comment:
I thought this will be called something more generic, like ```debug``` or ```inspect```. dump will become a configuration item then. In the future we can then add more configuration items to do more debug-sort of things.